packages/sources/src/addSourceWizard/hardcodedComponents/aws/arn.js

```diff
@@ -84,34 +84,20 @@ export const IAMPolicyDescription = () => {
             <TextListItem component={TextListItemVariants.li}><FormattedMessage id="wizard.CreateANewPolicyPastingTheFollowingContentIntoTheJsonTextBox" defaultMessage="Create a new policy, pasting the following content into the JSON text box." /></TextListItem>
             <ClipboardCopy isCode variant={ClipboardCopyVariant.expansion} className="pf-u-m-sm-on-sm" isReadOnly>
                 {JSON.stringify({
-                    Version: '2012-10-17',
-                    Statement: [
-                        {
-                            Sid: 'VisualEditor0',
-                            Effect: 'Allow',
-                            Action: [
-                                's3:Get*',
-                                's3:List*'
-                            ],
-                            Resource: [
-                                `arn:aws:s3:::${s3Bucket}`,
```

Comment:
s3Bucket si no longer used, so you can remove the formOptions from this component

Reply:
Right! I will fix it. Thanks